kgforge/core/archetypes/store.py

```diff
@@ -525,3 +529,39 @@ def replace(match: Match) -> str:
     if context.has_vocab():
         pfx = "\n".join([pfx, f"PREFIX : <{context.vocab}>"])
     return f"{pfx}\n{qr}"
+
+
+def resources_from_construct_query(data, context):
```

Comment:
All these was part of blue_brain_nexus store, and I made it a method in order to use it also in the sparql store.